Maquetado/index.html

```diff
@@ -1,21 +1,33 @@
 <!doctype html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <title>HTML & CSS Training</title>
-  <meta name="description" content="training">
-  <meta name="author" content="Wolox">
-  <link rel="stylesheet"  type="text/css" href="application.css">
-  <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
-</head>
-<body>
-  <div class="container">
-    <header>
-      <img src="assets/triangle.svg" alt="triangle">
-      <h3>Front-End Tools</h3>
-    </header>
-    <h1 class="title">Hello World</h1>
-  </div>
-</body>
+  <head>
+    <meta charset="utf-8">
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet"  type="text/css" href="application.css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <img src="assets/triangle.svg" alt="triangle">
+        <h3>Front-End Tools</h3>
+      </header>
+      <div class="tech">
+        <div class="base">
```

Comment:
Mmm i think one div Is enough. Try not ti wrap things in a div unless it's really necessary. Mayb hoy van merge all the Styles in one div

Reply:
<img width="1680" alt="screen shot 2018-11-26 at 9 11 46 am" src="https://user-images.githubusercontent.com/43754771/49019351-ba9d7780-f15b-11e8-8e33-acefb18e1106.png">

I made this option using a container that contains card.

Reply:
<img width="1680" alt="screen shot 2018-11-26 at 9 35 20 am" src="https://user-images.githubusercontent.com/43754771/49020538-adce5300-f15e-11e8-8876-c2ef4c81a6ab.png">

I made this option using a container that contains card.